meta-resin-common/recipes-core/balena-rollback/files/rollback-altboot.service

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 Resinio Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[Unit]
+Description=Run rollback-altboot to run rollback checks
```

Comment:
I would remove "Run" as systemd already prepends things like running/starting... stopping etc. "Ballena rollback checks" sounds more inspired to me.